### PR TITLE
Use `atomically` package instead of `write-file-atomic`

### PR DESCRIPTION
**Pull request by jurepetrovic**
Use atomically instead of writeFileAtomic. 

https://github.com/sindresorhus/electron-store/issues/103

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-	"funding": "https://github.com/sponsors/sindresorhus",
+	"repository": "jurepetrovic/conf-atomically",
+	"funding": "https://github.com/sponsors/jurepetrovic",
 	"author": {
-		"name": "Sindre Sorhus",
-		"email": "sindresorhus@gmail.com",
-		"url": "https://sindresorhus.com"
+		"name": "Jure Petrovic",
+		"email": "jurep2006@gmail.com",
+		"url": "https://www.enerco.si"
```
Comment (papb) [inline package.json:11]: Hello, please undo these changes

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # conf [![Build Status](https://travis-ci.com/sindresorhus/conf.svg?branch=master)](https://travis-ci.com/github/sindresorhus/conf)
 
+
+## FORK
+
+This fork uses package atomically to prevent EPERM errors on Windows machines.
+
+
+
+
```
Comment (papb) [inline readme.md:10]: Also undo this change

File: source/index.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ import dotProp = require('dot-prop');
 import makeDir = require('make-dir');
 import pkgUp = require('pkg-up');
 import envPaths = require('env-paths');
-import writeFileAtomic = require('write-file-atomic');
+import writeFileAtomic = require('atomically');
```
Comment (papb) [inline source/index.ts:10]:
```suggestion
import atomically = require('atomically');
```

File: source/index.ts [hunk 4]
```diff
@@ -451,7 +451,7 @@ class Conf<T extends Record<string, any> = Record<string, unknown>> implements I
 			fs.writeFileSync(this.path, data);
 		} else {
 			try {
-				writeFileAtomic.sync(this.path, data);
+				writeFileAtomic.writeFileSync(this.path, data);
```
Comment (papb) [inline source/index.ts:454]:
```suggestion
				atomically.writeFileSync(this.path, data);
```

File: package.json (hunk 1 above)
Comment (jurepetrovic) [reply package.json:11]: sorry, I needed working npm package in npmjs....

**Comment by jurepetrovic** [comment]
Ok, should be fine now.

**Comment by jurepetrovic** [comment]
Please also release new NPM package if ok

**Comment by sindresorhus** [comment]
@jurepetrovic Tip for the future: Never do PRs from `master` branch.

**Comment by jurepetrovic** [comment]
@sindresorhus yes, figured that one out the hard way. Thanks ;-)
